drizztsaga/drizztsaga.tp2

```diff
@@ -355,19 +359,47 @@ ACTION_IF GAME_IS ~BGT~ BEGIN
 	BUT_ONLY
 END
 
+
 ///////////////////////////////////////////////////////////////////////////////////////////////
-//// New encounter: Valen in the Undercity - an optional component 			  /////////////////
+//// Nerf items: items are now "only" +3 and undroppable - an optional component/////////////////
```

Comment:
The change seems to be unrelated to the current Pull Request. Could you please revert the item nerfing and leave only Valen-related changes? Thanks.

Reply:
yes, that is a mistake